frontend/src/individual_certificate.tsx

```diff
@@ -135,6 +135,28 @@ export function IndividualCertificate({
           "error"
         );
   }, [id, user]);
+
+  /**
+   * The API sends the birth date in a format like this:
+   * "Thu, 08 Jul 2021 00:00:00 GMT".
+   * This function turns it into a format like this: "YYYY-MM-DD"
+   */
+  React.useEffect(() => {
+    const formatDate = (fullDate: string) => {
+      const date = new Date(fullDate).toISOString();
+      const dateString = date.split("T")[0];
+      return dateString;
+    };
+    if (
+      // check if the birth date is still in the wrong format,
+      // i.e. has more than 10 characters
+      individual?.birth_date &&
+      individual.birth_date.length > 10
```

Comment:
I think this is no longer needed. If we reach year 10000 this would stop working!